public/app/features/alerting/unified/hooks/ruleGroup/usePauseAlertRule.test.tsx

```diff
@@ -1,17 +1,17 @@
 import userEvent from '@testing-library/user-event';
 import { HttpResponse } from 'msw';
 import { render } from 'test/test-utils';
-import { byText, byRole } from 'testing-library-selector';
+import { byRole, byText } from 'testing-library-selector';
```

Comment:
Opened a PR to make them all consistently ordered 😁 
https://github.com/grafana/grafana/pull/97688